Sources/SwiftInspectorVisitors/Tests/NestableTypeVisitorSpec.swift

```diff
@@ -180,6 +200,26 @@ final class NestableTypeVisitorSpec: QuickSpec {
               expect(structInfo?.name) == "SomeStruct"
               expect(structInfo?.genericParameters.map { $0.name }) == ["A", "B"]
             }
+
+            context("that is constrained with a where clause") {
```

Comment:
"with one that is..."?

Reply:
(since the parent context is "with two generic parameters")

Reply:
Same comment below.

Reply:
87bf5a3